opm/simulators/wells/StandardWell_impl.hpp

```diff
@@ -1434,7 +1434,6 @@ namespace Opm
             well_state_copy.wellRates(this->index_of_well_)[phase]
                     = sign * ws.well_potentials[phase];
         }
-        well_copy.calculateExplicitQuantities(ebosSimulator, well_state_copy, deferred_logger);
```

Comment:
The alternative is that the primary variables can be updated, while the connection pressure drop should not be updated since there is no consistent perforation rate here.

Reply:
updated along this line.